app/root.tsx

```diff
@@ -36,6 +37,8 @@ export async function loader({ request, context }: LoaderArgs) {
   });
 }
 
+addReconcilerJob();
```

Comment:
This seems like the wrong place to do this.  I get that Remix is annoying with this stuff, and I've struggled a lot to find an optimal solution.

It feels like something that could live in `server.ts`, despite the fact that I ripped all that code out of there.  I'm not sure what to do.  If you want to leave this here for now, and we fix it later, that's also doable.  I don't want to block this work on something stupid.

For follow-up, I want to figure out how to move the bulk of the queue stuff out of the `app/` and have it be associated with the server anyway.

Reply:
Yes, this gets called multiple times during development, I simply could not find a better place (and not for the lack of trying!) Also all other queues are "implicitly" initialized, just because of the file import. Since This piece does not get called from the UI or anywhere else, and if I only don an `import 'filename'` it does not work at all (code in the file is not executed)

I agree, we should do a follow-up